### PR TITLE
quick compilation fix for scheduler sweat tests

### LINKED ISSUE #32223: scheduler: test-performance.sh compile error.
_Problem_

Looks like the test-performance.sh utility doesn't work because of a compile change in test/integration/framework `DeleteAllEtcdKeys` ?

_Solution_ 

Reimplement DeleteAllEtcdKeys ? 

_Details_

```
➜  perf git:(master) ✗ ./test-performance.sh 
~/Development/kubernetes/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/component/scheduler/perf ~/Development/kubernetes/src/k8s.io/kubernetes/_output/local/go/src/k8s.io/kubernetes/test/component/scheduler/perf
etcd --advertise-client-urls http://127.0.0.1:2379 --data-dir /var/folders/nc/49qzdyt931bg1hgpc2bz277r0000gn/T/test-etcd.XXXXXX.bOkDF0Nl --listen-client-urls http://127.0.0.1:2379 --debug > "/dev/null" 2>/dev/null
Waiting for etcd to come up.
+++ [0907 15:57:58] On try 2, etcd: : http://127.0.0.1:2379
{"action":"set","node":{"key":"/_test","value":"","modifiedIndex":4,"createdIndex":4}}
+++ [0907 15:57:58] performance test start
```

then a compile error appears...

```
# k8s.io/kubernetes/test/component/scheduler/perf
./util.go:47: undefined: "k8s.io/kubernetes/test/integration/framework".DeleteAllEtcdKeys
FAIL    k8s.io/kubernetes/test/component/scheduler/perf [build failed]
```

### PR DESCRIPTION
**Pull request by jayunit100**
fixes #32223 
needs feedback from @liggitt and @kubernetes/sig-scheduling  before merge.

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/32384)

### CODE REVIEW

**Comment by jayunit100** [comment]
```
333s    rate: 86        total: 28932
334s    rate: 58        total: 28990
335s    rate: 84        total: 29074
336s    rate: 82        total: 29156
337s    rate: 85        total: 29241
338s    rate: 86        total: 29327
339s    rate: 72        total: 29399
340s    rate: 74        total: 29473
341s    rate: 82        total: 29555
342s    rate: 85        total: 29640
E0909 14:18:37.090594   24749 event.go:208] Unable to write event: 'Post http://127.0.0.1:38460/api/v1/namespaces/default/events: dial tcp 127.0.0.1:38460: getsockopt: connection refused' (may retry after sleeping)
343s    rate: 84        total: 29724
344s    rate: 84        total: 29808
345s    rate: 41        total: 29849
346s    rate: 65        total: 29914
347s    rate: 78        total: 29992
348s    rate: 8 total: 30000
```

File: pkg/util/certificates/csr.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func NewCertificateRequest(privateKey interface{}, subject *pkix.Name, dnsSANs [
 	switch privateKey := privateKey.(type) {
 	case *ecdsa.PrivateKey:
 		switch privateKey.Curve {
-		case elliptic.P224(), elliptic.P256():
+		case elliptic.P256():
```
Comment (liggitt) [inline pkg/util/certificates/csr.go]: undo this

File: pkg/util/certificates/csr.go [hunk 2]
```diff
@@ -93,7 +93,7 @@ func NewCertificateRequest(privateKey interface{}, subject *pkix.Name, dnsSANs [
 	switch privateKey := privateKey.(type) {
 	case *ecdsa.PrivateKey:
 		switch privateKey.Curve {
-		case elliptic.P224(), elliptic.P256():
```
Comment (ingvagabund) [inline pkg/util/certificates/csr.go]: On Fedora ``P.224`` is removed intentionally [1]. Does not have to apply for other distributions.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1038683

**Comment by jayunit100** [comment]
Using the gimmee go or another go that has the p224 stuff worksaround the missing curve issues.

```
unset GOOS
unset GOARCH
export GOROOT="/root/.gimme/versions/go1.6.3.linux.amd64"
export PATH="/root/.gimme/versions/go1.6.3.linux.amd64/bin:${PATH}"
go version >&2
```

**Comment by jayunit100** [comment]
@k8s-bot test this please

**Comment by jayunit100** [comment]
force pushed a commit change, if `unexpected creation success. The following IPs exist: []string{"192.168.0.1", "192.168.0.1", "192.168.0.2"}. It should only be possible to allocate 2 IP addresses in this cluster.` then , somehow, this is a breaking change.  i hope its just a flake though :)

**Comment by liggitt** [comment]
could that have been caused by running a test that expected to run on a clean etcd on top of possibly existing data?

**Comment by jayunit100** [comment]
I think it was a flake.  It seems to be ok now....?
The old etc cleaning function didn't compile anyways

**Comment by davidopp** [comment]
@wojtek-t

**Comment by jayunit100** [comment]
This was subsumed  in the broader PR to automate integration validation for scheduler-perf

**Pull request closed without merging**